packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -8449,6 +8449,23 @@ The application provisioning connection profile is used to configure the method
 | authScheme              | Defines the method of authentication     | `TOKEN` | FALSE    | FALSE  | FALSE    |
 | token            | Token used to authenticate with application      | String | FALSE    | FALSE   | FALSE    |
 
+#### OAuth 2.0 based Provisioning Connection Profile example
+```json
+{
+    "profile": {
+        "authScheme": "OAUTH2",
+        "clientId": "${clientId}"
+    }
+}
+```
+
+#### OAuth 2.0 based Provisioning Connection Profile properties
```

Comment:
```suggestion
#### OAuth 2.0-based Provisioning Connection Profile properties
```
Maybe add `<ApiLifecycle access="ea" />` below this header to apply EA flag to entire table?